packages/communication-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/communication-react",
-  "version": "1.14.0-beta.2",
+  "version": "1.14.1-beta.0",
```

Comment:
Did this bump correctly? or should it be `1.15.0-beta.0`?

Reply:
I had the same question.

Reply:
Both will work and only impact the alpha release versions. But from a previous discussion we want new releases to always be minor unless they are hotfixes in which case yes we should fix this to bump minor and not patch.
Filed a bug to capture this for future work, this shouldn't block this PR however [Bug 3652130](https://skype.visualstudio.com/SPOOL/_workitems/edit/3652130): [Web] Stable prerelease bump should bump minor version and not patch version